src/views/Card.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import { Container, Header, Content, Card as NativeBaseCard, CardItem, Body, Title } from 'native-base';
+import FlipCard from 'react-native-flip-card'
+
+export default class Card extends Component {
+  render() {
+    return (
+      <Container>
+        <Header> 
+          <Body>
+						<Title>Virtual Cards</Title>
+          </Body>
+        </Header>
+        <Content scrollEnabled={false}>
+          <FlipCard
+            friction={6}
+            perspective={1000}
+            flipHorizontal={true}
+            flipVertical={false}
+            flip={false}
+            clickable={true}
```

Comment:
Can't we add like default values for those properties so we don't have to specify them every time?